hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
+
+setup_cluster() {
+    git clone https://github.com/k8snetworkplumbingwg/multus-cni/
+    trap "popd && cleanup" EXIT
+    pushd multus-cni/e2e
+    ./get_tools.sh
+    CNI_VERSION="$CNI_VERSION" ./generate_yamls.sh
+    OCI_BIN="$OCI_BIN" ./setup_cluster.sh
+    trap - EXIT
+    popd
+}
+
+push_local_image() {
+    OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" make manifests
+    OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" make img-build
+    "$OCI_BIN" push "$IMG_REGISTRY/multus-dynamic-networks-controller:latest"
+}
+
+cleanup() {
+    rm -rf multus-cni
+    git checkout -- manifests/
```

Comment:
can we checkout everything instead of manifest ?
on the otherhand do we really care ?
it just runs in a container of git actions

Reply:
I also use this locally :)

Reply:
maybe worth to have some `make test-cleanup` with that command ?